src/tutorials/Mutable-File-System/02.md

```diff
@@ -1,5 +1,26 @@
 
-Within each exercise, you'll see that you can upload from your computer either by dragging and dropping or selecting from your file explorer. If you look closely at the `run` function in the code editor, you'll notice that it now takes an argument `files`. When you upload files from your computer, we'll make sure they're passed into the function as the `files` array. As long as you don't refresh your browser, these files will remain accessible for the next lesson in the tutorial, but you'll also have the option to upload different files to work with for each lesson.
+We're creating your IPFS node behind the scenes so you can focus on the content of our lessons, but eventually you'll need to learn to host your own node locally. When you're ready to experiment, you can find instructions for [installing IPFS](https://docs.ipfs.io/introduction/install/) and [initializing your node](https://docs.ipfs.io/introduction/usage/) in our docs.
 
-To practice, let's upload one or more files from your computer and create a console log to see what's been received by the browser as the `files` array.
+As mentioned previously, methods associated with the Mutable File System are part of the Files API, so they'll take the format of `ipfs.files.someMethod()`. Let's take a look at a simple method you can start using even before you've added any files to your IPFS node.
+
+## Exploring your IPFS node with `ipfs.files.stat`
+When working with your IPFS node, you'll often want to check the status of a file or directory. You can do this with `ipfs.files.stat`, passing in the path you'd like to check on.
+
+For example, to check the status of a directory called `stuff` located within our root directory ( `/` ), we could call the method like so:
+
+`await ipfs.files.stat('/stuff/')`
+
+This method returns an object with some essential data about our file or directory:
+
+* **hash** (a string with the cryptographic hash)
+* **size** (an integer with the file or folder size in Bytes)
+* **cumulativeSize** (an integer with the size of the DAGNodes making up the file in Bytes)
+* **type** (a string that can be either `directory` or `file`)
+* **blocks** (if type is `directory`, this is the number of files in the directory; if type is `file`, it's the number of blocks that make up the file)
+* **withLocality** (a boolean to indicate if locality information are present)
+* **local** (a boolean to indicate if the queried dag is fully present locally)
+* **sizeLocal** (an integer indicating the cumulative size of the data present locally)
+
+It's important to note that you can `stat` your IPFS node even when you don't have anything in it yet. Even empty nodes have CIDs (hashes) that represent their contents.
```

Comment:
```suggestion
It's important to note that you can `stat` your IPFS node even when you don't have anything in it yet. Even empty nodes have a CID (hash).
```

Reply:
There are two separate issues here - one is the plural. If you prefer it without the plural then we can switch to "Even an empty node has a CID (hash)" so the subject is singular also. 

I'm assuming your objection to "that represent their contents" was that this empty node doesn't have any contents, which is precisely what I was trying to call out. I want to remind people that the CID is directly connected to the data - or lack thereof. 

Would this make sense to you? 
"It's important to note that you can `stat` your IPFS node even when you don't have anything in it yet. Even an empty node has a CID (hash) that represents its (non-existent) contents." (or "contents (or lack thereof)")

I'm open to better suggestions!